openshift/scripts/oc_provision_matomo_backup_pvc.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh -l
 #
+# TODO: Delete this file! Redundant in OCP4
```

Comment:
This one too?

Reply:
Oh you are going to delete them when we are done with OCP3 I see.

Reply:
Yep. Same deal - it's running in pathfinder.